fix: Be able to load meteor apps from Shopware bundles (#10530)
shopware/platform: changelog/_unreleased/2025-06-13-allow-admin-scripts-provided-by-bundles.md

---
title: Allow admininistration scripts being provided by bundles
author: Joshua Behrens
author_email: code@joshua-behrens.de
author_github: @JoshuaBehrens
---
# Core
* Changed focus in administration asset path detection from `\Shopware\Core\Framework\Plugin` to `\Shopware\Core\Framework\Bundle` to be able to load meteor apps from bundles  

> * Fixed usage of superfluous bundle suffix in administration asset path building in `\Shopware\Core\Framework\Api\Controller\InfoController`

Reviewer: ```suggestion
* Changed bundle suffix in administration asset path building in `\Shopware\Core\Framework\Api\Controller\InfoController` by removing bundle suffix
```
